src/relay/pass/let_list.h

```diff
@@ -108,6 +111,7 @@ class LetList {
 
  private:
   std::vector<std::pair<Var, Expr> > lets_;
+  bool used_ = false;
```

Comment:
Do we really need this flag? I mean, is it used for anything other than sanity check?

Reply:
It is sanity check only. It *does* caught an error(which will otherwise be silent and generate incorrect Expr) when I code, so I think we should keep it.

Reply:
Okie, good to know